README.md

```diff
@@ -26,6 +26,36 @@ Update example: After changing the Docker image inside the operator's configurat
 it manages and updates them with the new Docker image; afterwards, all pods from each StatefulSet are killed one by one
 and the replacements are spawned automatically by each StatefulSet with the new Docker image. This is called the Rolling update.
 
+
+## Quickstart 
+
+Prerequisites: [minikube](https://github.com/kubernetes/minikube/releases) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-native-package-management)
+
+### Local execution
+
+```bash
+git clone https://github.com/zalando-incubator/postgres-operator.git
+cd postgres-operator
+
+minikube start
+kubectl config set-context minikube
+
+kubectl create -f manifests/configmap.yaml         # operator config
```

Comment:
maybe `apply` is better?

Reply:
The `apply` and `create` commands seem to come from [different ways](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/overview/#management-techniques) of managing k8s objects. I've used create for now to:
 (a) explicitly show the order of resource creation 
 (b) for consistency with the rest of the docs that use `create` as well